web/src/input_pill.ts

```diff
@@ -29,7 +29,7 @@ type InputPillCreateOptions<ItemType> = {
     ) => ItemType | undefined;
     get_text_from_item: (item: ItemType) => string;
     get_display_value_from_item: (item: ItemType) => string;
-    generate_pill_html?: (item: ItemType) => string;
+    generate_pill_html?: (item: ItemType, disabled?: boolean) => string;
```

Comment:
This first commit looks good even if we're not using it now, since it has a test.

Reply:
In particular, we may end up using it to display inherited permissions in some group settings inputs in some future viewing mode.